hack/bootstrap-cluster.sh

```diff
@@ -2,6 +2,14 @@
 
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
 
+OC_WHOAMI="$(oc whoami)"
+if [ "$OC_WHOAMI" != "kubeadmin" ]; then
```

Comment:
I would not connect this with specific user, any user with cluster-admin role can bootstrap the cluster. You can check if the user can do anything on cluster using `oc auth can-i '*' '*' --all-namespaces`. If not then I prefer to stop script with proper message that user should login as user with cluster-admin role and run the script again.

Reply:
Fixed. I briefly mention the kubeadmin user in the help message so that newcomers don't have to guess which user they should be connected with.